Helper/ShipmentOptions.php

```diff
@@ -232,8 +232,7 @@ public static function getValueOfOptionWhenSet(array $options, string $key): ?bo
      */
     public function hasLargeFormat(): bool
```

Comment:
Wordt deze functie vaker dan 1x gebruikt? Anders kan hij net zo goed helemaal weg

Reply:
Ja deze wordt in ShipmentOptions zelf een keer gebruikt, maar ook in TrackAndTraceHolder